src/v/serde/test/BUILD

```diff
@@ -0,0 +1,48 @@
+load("//bazel:test.bzl", "redpanda_cc_btest", "redpanda_test_cc_library")
+
+redpanda_test_cc_library(
+    name = "serde_test_def",
+    defines = [
+        "SERDE_TEST",
+    ],
+    visibility = ["//visibility:private"],
+)
+
+redpanda_cc_btest(
```

Comment:
I think it is fine if you want to plumb `defines` parameter through `redpanda_cc_btest` macro so that we don't have to define an empty library to inject the definition.

Reply:
oh awesome! i'll do just that.